Classes/MatchHistory.lua

```diff
@@ -1,255 +1,255 @@
-            local healColumn = columns[3]
-            local killColumn = columns[4]
-
-            -- If i am the last player in my team show the separator line, otherwise hide
-            row:SetSeparatorLine(playerIndex == playerTeamSize and not self.isSoloShuffle)
-
-            playerColumn.text:SetText(f.class(player.name, player.class))
-            dmgColumn.text:SetText(PvPAssistant.UTIL:FormatDamageNumber(player.scoreData.damageDone))
-            healColumn.text:SetText(PvPAssistant.UTIL:FormatDamageNumber(player.scoreData.healingDone))
-            killColumn.text:SetText(PvPAssistant.UTIL:FormatDamageNumber(player.scoreData.killingBlows))
-        end)
-    end
-
-    playerList:UpdateDisplay()
-end
-
----@class PvPAssistant.MatchHistory.Serialized
----@field timestamp number
----@field mapInfo InstanceInfo
----@field isArena boolean
----@field isBattleground boolean
----@field isSoloShuffle boolean
----@field playerTeam PvPAssistant.Team
----@field enemyTeam PvPAssistant.Team
----@field duration number
----@field isRated boolean
----@field pvpMode PvPAssistant.Const.PVPModes
----@field win boolean
----@field season number
----@field player PvPAssistant.Player
----@field pvpMatchState Enum.PvPMatchState
-
----@return PvPAssistant.MatchHistory.Serialized
-function PvPAssistant.MatchHistory:Serialize()
-    ---@type PvPAssistant.MatchHistory.Serialized
-    local serialized = {
-        timestamp = self.timestamp,
-        mapInfo = self.mapInfo,
-        isArena = self.isArena,
-        isBattleground = self.isBattleground,
-        isSoloShuffle = self.isSoloShuffle,
-        playerTeam = self.playerTeam,
-        enemyTeam = self.enemyTeam,
-        duration = self.duration,
-        isRated = self.isRated,
-        pvpMode = self.pvpMode,
-        win = self.win,
-        season = self.season,
-        player = self.player,
-        pvpMatchState = self.pvpMatchState,
-    }
-
-    return serialized
-end
-
----@param serializedData PvPAssistant.MatchHistory.Serialized
----@return PvPAssistant.MatchHistory matchHistory
-function PvPAssistant.MatchHistory:Deserialize(serializedData)
-    local matchHistory = PvPAssistant.MatchHistory()
-    matchHistory.timestamp = serializedData.timestamp
-    matchHistory.mapInfo = serializedData.mapInfo
-    matchHistory.isArena = serializedData.isArena
-    matchHistory.isBattleground = serializedData.isBattleground
-    matchHistory.isSoloShuffle = serializedData.isSoloShuffle
-    matchHistory.playerTeam = serializedData.playerTeam
-    matchHistory.enemyTeam = serializedData.enemyTeam
-    matchHistory.duration = serializedData.duration
-    matchHistory.isRated = serializedData.isRated
-    matchHistory.pvpMode = serializedData.pvpMode
-    matchHistory.win = serializedData.win
-    matchHistory.season = serializedData.season
-    matchHistory.player = serializedData.player
-    matchHistory.pvpMatchState = serializedData.pvpMatchState
-    return matchHistory
-end
+---@class PvPAssistant
```

Comment:
Could you check your auto format config? Tabs should be 4 spaces (I think) Ill check tmr to be sure

Reply:
I'm using 4 spaces. 🤔